docs/gettingStarted/REACT.md

```diff
@@ -211,6 +211,14 @@ For further details on how to customize the configuration file, check out the [C
 
 If you run into any issues while running the `zeplin connect` command, make sure that you have the React plugin installed.
 
+```sh
+npm list -g --depth 0 @zeplin/cli-connect-react-plugin
+```
+
+### "I cannot see code snippets for my components on Zeplin"
```

Comment:
We may mention a problem with pure styled-components. Patrick had reached us before about this issue for one of our users on beta.

I found out that 2 weeks ago somehow has published a 3rd party resolver to handle the problem. Opened a task for me, let's see if we can deal with it or mention the problem with its workaround solution.